src/common/decorators/validation.ts

```diff
@@ -166,3 +169,66 @@ export function IsTxTagOrModuleName(validationOptions?: ValidationOptions) {
     });
   };
 }
+
+@ValidatorConstraint({ async: false })
+export class IsNotSiblingOfConstraint implements ValidatorConstraintInterface {
```

Comment:
```suggestion
 class IsNotSiblingOfConstraint implements ValidatorConstraintInterface {
```
does this need to be exported?